nuxeo-java-client/src/main/java/org/nuxeo/client/objects/blob/Blobs.java

```diff
@@ -40,27 +37,4 @@ public Blobs(List<? extends Blob> blobs) {
         super(EntityTypes.BLOBS, blobs);
     }
 
-    public List<Blob> getBlobs() {
-        return getEntries();
-    }
```

Comment:
Not marked as deprecated?

Reply:
Indeed, this was a forgotten deprecation and as it is a getter we can found in other classes, I though it was ok to remove it.